dicom-archive/database_config_template.py

```diff
@@ -6,4 +6,27 @@
     'passwd'  : 'DBPASS',
     'database': 'DBNAME',
     'port'    : ''
-}
\ No newline at end of file
+}
+
+def get_subject_ids(db, dicom_value=None, scanner_id=None):
+
+    subject_id_dict = {}
+
+    imaging = Imaging(db, False)
```

Comment:
modify to add the import otherwise get the following error:
```
Traceback (most recent call last):
  File "python/dicom_archive_validation.py", line 316, in <module>
    main()
  File "python/dicom_archive_validation.py", line 82, in main
    validate_dicom_archive(config_file, tarchive_path, upload_id, verbose)
  File "python/dicom_archive_validation.py", line 258, in validate_dicom_archive
    site_dict = imaging_obj.determine_study_center(tarchive_info_dict)
  File "/data/ccna/bin/mri-test/python/lib/imaging.py", line 434, in determine_study_center
    subject_id_dict = self.determine_subject_ids(tarchive_info_dict)
  File "/data/ccna/bin/mri-test/python/lib/imaging.py", line 351, in determine_subject_ids
    subject_id_dict = self.config_file.get_subject_ids(self.db, dicom_value, scanner_id)
  File "/data/ccna/bin/mri-test/dicom-archive/.loris_mri/database_config.py", line 15, in get_subject_ids
    imaging = Imaging(db, False)
NameError: name 'Imaging' is not defined
```

Reply:
+ re import